workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -93,15 +93,31 @@ public async Task Connect(string workerType, ILogDispatcher logger)
                 ConnectionDelegate connectionDelegate;
                 if (ShouldUseLocator())
                 {
+                    var config = GetLocatorConfig(workerType);
+                    if (!config.Validate(out var errorMessage))
```

Comment:
This is here to handle an edge case where we neither use the login token flow nor the steam flow because neither command line arguments were specified. I wanted to handle the problem on this level because 
- we are not using the Validate() function outside of tests for some reason
- that way this issue doesn't need to be addressed at the `SteamClientWorkerConnector` level.